cl_sii/extras/dj_form_fields.py

```diff
@@ -0,0 +1,72 @@
+"""
+cl_sii "extras" / Django form fields.
+
+"""
+try:
+    import django
+except ImportError as exc:  # pragma: no cover
+    raise ImportError("Package 'Django' is required to use this module.") from exc
+
+from typing import Any, Optional
+
+import django.core.exceptions
+import django.db.models
+import django.db.models.fields
+import django.forms
+from django.utils.translation import gettext_lazy as _
+
+from cl_sii.rut import Rut
+
+
+class RutField(django.forms.CharField):
+
+    """
+    Django form field for RUT.
+
+    * Python data type: :class:`cl_sii.rut.Rut`
+
+    .. seealso::
+        :class:`.dj_model_fields.RutField`
+
+    """
+
+    default_error_messages = {
+        'invalid': _('Enter a valid RUT.'),
```

Comment:
I don't know whether we'll want to have support for translation or not, or if it's just a bonus. @yaselc @jtrh your call

Reply:
> I don't know whether we'll want to have support for translation or not, or if it's just a bonus.

I think that someday we should, at least for Spanish

Regarding the change itself, if `_('Enter a valid RUT.')` doesn't cause any additional work to our library users (e.g. by forcing them to execute `django-admin makemessages` and `django-admin compilemessages`), then I think it's fine.